visual-regression-preflight.js

```diff
@@ -0,0 +1,19 @@
+const fetch = require('node-fetch');
+
+const PULL_REQUEST_ID = process.env.CIRCLE_PULL_REQUEST.split('/').pop();
+const LABELS_TO_STOP = ['wip', 'do not merge'];
+
+(async () => {
+    const request = fetch(`https://api.github.com/repos/justeat/fozzie-components/pulls/${PULL_REQUEST_ID}`);
+    const response = await request;
+    const json = await response.json();
+    const details = {
+        draft: json.draft,
+        labels: json.labels.map(l => l.name)
+    };
+
+
+    // If PR is draft or has given labels, skip visual regression tests to save credits
+    const shouldRunTests = !details.draft && !LABELS_TO_STOP.some(label => details.labels.includes(label));
+    console.log(shouldRunTests);
```

Comment:
should this val be returned?

Reply:
This PR isn't quite ready for review, but I'll answer anyway 😄 
This seems to be the way to "return" a value from a node script so that it can be read by the shell that runs it, see here:
https://github.com/justeat/fozzie-components/pull/1304/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R227

There may be better ways but this is just a spike atm

Reply:
Ah cheers! I was being nosey ;)